jenkins/resource_jenkins_credential_secret_file_test.go

```diff
@@ -0,0 +1,173 @@
+package jenkins
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	jenkins "github.com/bndr/gojenkins"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccJenkinsCredentialSecretFile_basic(t *testing.T) {
+	var cred jenkins.FileCredentials
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckJenkinsCredentialSecretFileDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: `
+				resource jenkins_credential_secret_file foo {
+				  name = "test-secret-file"
+				  filename = "secret.txt"             
+				  secretbytes = "VGhpcyBpcyBhIHRlc3Qu"
+				}`,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("jenkins_credential_secret_file.foo", "id", "/test-secret-file"),
+					testAccCheckJenkinsCredentialSecretFileExists("jenkins_credential_secret_file.foo", &cred),
+				),
+			},
+			{
+				// Update by adding description
+				Config: `
+				resource jenkins_credential_secret_file foo {
+				  name = "test-secret-file"
+				  description = "new-description"
+                                  filename = "secret.txt"             
+                                  secretbytes = "VGhpcyBpcyBhIHRlc3Qu"
```

Comment:
Maybe update the contents of the `secretbytes` property to prove out that the provider supports update operations for the value?

Reply:
done